### PR TITLE
Make all of Debug Console Static

### PR DESCRIPTION
**Pull request by kylekrol**
### Summary of changes

Make `debug_console` an entirely static class. From a architecture perspective, this is the most natural as we only have a single serial port to the host computer.

### Testing

HITLs were run on the EDU and multiple HOOTLs were run locally.

### CODE REVIEW

File: src/common/debug_console.cpp [hunk 1]
```diff
@@ -5,79 +5,121 @@
...
-    unsigned int elapsed_time = current_time - _start_time;
-    return elapsed_time;
+    unsigned int const current_time = millis();
+
+    /* Reset the start time if Serial is unconnected. We do this so that the
+     * logging utility on the computer can always produce a correct timestamp.
+     */
+    if (!Serial) start_time = current_time;
+
+    return current_time - start_time;
```
Comment (tanishqaggarwal) [inline src/common/debug_console.cpp:89]: God this was such terrible code haha thanks

**Review by tanishqaggarwal — COMMENTED** [review]
Changes seem benign enough.

**Review by tanishqaggarwal — APPROVED** [review]

**Review by shihaocao — APPROVED** [review]
big

File: src/common/debug_console.cpp (hunk 1 above)
Comment (kylekrol) [reply src/common/debug_console.cpp:89]: Hey dude it was the OG PAN days when none of us new what C++ was lolll And it also got us this far!

**Pull request merged**